src/vis/density/DensityVis.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import { merge, uniqueId } from 'lodash';
+import { useMemo } from 'react';
+import { VisColumn, IVisConfig, IDensityConfig, EScatterSelectSettings } from '../interfaces';
+import { DensityVisSidebar } from './DensityVisSidebar';
+// eslint-disable-next-line import/no-cycle
```

Comment:
```suggestion
```
Can we remove this line? Locally my ESLint was not complaining when I removed it.